### ISSUE TITLE
g2o how to do rotation only or translation only?

### ISSUE THREAD

**Issue by miaoiyy** [RainerKuemmerle/g2o#186]
Hi, I am using g2o icp to alignment two point clouds. Due to the limitation of the sensor, I do not want to do the rotation and translation both, or even I only want the vertices to move in x and y directions and stay still in the z direction.  Is there any way to do it? now I am giving the larger number in information matrix for the one I do not want to change. But I am not sure whether it is right or not.

**Comment by mojovski** [comment]
Yes, but you will have to write your own derivatives and error functions.
